skyvern/core/script_generations/skyvern_page.py

```diff
@@ -307,13 +317,18 @@ async def _update_action_reasoning(
     async def _create_action_after_execution(
         self,
         action_type: ActionType,
-        intention: str = "",
+        prompt: str = "",
         status: ActionStatus = ActionStatus.pending,
         data: str | dict[str, Any] = "",
         kwargs: dict[str, Any] | None = None,
         call_result: Any | None = None,
+        intention: str = "",  # backward compatibility
```

Comment:
i don't think we need both intention and prompt here since the function that calls _create_action_after_execution already merged the two